### ISSUE TITLE
Données Covid + dimension actuarielle

### ISSUE THREAD

**Issue by JeanClaudeR** [natR/Challenge-Data-Viz-2021#2]
### **Données publiques proposées à juin 2021**

données publiques d'hospi en retirant la variable Décès :
https://www.data.gouv.fr/fr/datasets/donnees-hospitalieres-relatives-a-lepidemie-de-covid-19/

IMPORTANT
Les analyses études graphiques sont purement actuarielles et scientifiques.
Ces éléments sont à inscrire dans le réglement et peut être en communication ? à voir avec Philippe T et Samuel pour accord expresse de leur part.

# Contexte
Ticket de travail pour trouver les données pertinentes et dans une certaine mesure les décrire.

# Résumé de la discussion
- Données Covid type John Hopkins ou "Our World in data"
- Données de surm.... https://github.com/natR/Challenge-Data-Viz-2021/issues/2#issuecomment-814846988
- CR Réunion Innovation 7/4/21 https://github.com/natR/Challenge-Data-Viz-2021/issues/2#issuecomment-815530390 : recherche durée hospitalisation
- Obliger a n'utiliser que des données librement accessible
- Données sur les traces de covid dans les égouts https://github.com/natR/Challenge-Data-Viz-2021/issues/2#issuecomment-848555639

**Comment by JeanClaudeR** [comment]
Données de surmortalité Covid + analyses INSEE et EuroMOMO
https://www.insee.fr/fr/statistiques/4637552?sommaire=4487854
https://www.insee.fr/fr/statistiques/4487854
https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2020.25.34.2001485?crawler=true
https://www.euromomo.eu/how-it-works/reports-and-publications/

**Comment by natR** [comment]
### **Suite commission Innovation du 7/04/2021**

Message de Florence 
Suggestion d'Anani Olympio : 
Travailler dans le Challenge Visualisation sur la durée d’hospitalisation Covid
Est-ce que les données sont disponibles: âge, sexe, région, sortie par décès?

Si on part du début de l’épidémie, ce doit être intéressant.

Est-ce un sujet possible?

**Comment by JeanClaudeR** [comment]
Finalement, on peut imaginer un thème global : Covid-19 
Puis décliner les sujets possibles : 
- sur la base des données de mortalité européennes
-  sur la base des données d'hospitalisation
Du coup, on peut indiquer quelques données de référence et laisser les participants rajouter celles qu'ils veulent tant qu'ils sont capables de raconter une bonne histoire

**Comment by FlorianPons** [comment]
Je suis d'accord avec Jean Claude, de plus il faut imposer de n'utiliser que des données librement accessibles.

**Comment by moilerat** [comment]
Données Obépine des égouts : https://solidarites-sante.gouv.fr/actualites/presse/communiques-de-presse/article/mise-a-disposition-en-open-data-des-donnees-de-surveillance-microbiologique-du

**Comment by MamyFlo** [comment]
Bonjour,

J’attire simplement l’attention sur le fait que l’Institut des actuaires souhaite éviter toute polémique relative à la COVID, en particulier à la réalité d’une surmortalité et aux causes de cette surmortalité.
Si on baisse le champ ouvert, je crois que l’Institut des actuaires demandera à avoir un droit de veto sur les décisions du jury afin d’éviter d’attribuer un prix à un résultat qui pourrait faire polémique.

Florence Picard

**Comment by natR** [comment]
Je trouve intéressante la proposition de Victor sur les données d'Obépine, qu'en dites vous ?

Nous restons dans le domaine de la Santé, prédire la diffusion du virus via les eaux usées me semblent un bon compromis.

**Comment by MamyFlo** [comment]
Confidentiel

Sujet qui fait débat chez les scientifiques car selon qu’il a plu ou fait chaud, un même degré de contamination n’est pas traduit de la même façon dans les eaux usées
Sujet polémique et politique également. Le FN a critiqué très tôt le gouvernement sur ce point

Florence
